### DISCUSSION TITLE
Intermittent Watermarks

### DISCUSSION THREAD

**Discussion by avalonMMXXII** [chrisbenincasa/tunarr#427] [Ideas]
Someone had suggested this awhile back and I know ETV has this feature, but Intermittent watermarks would be nice. Or even the ability to add multiple watermarks. Is this something under consideration for Tunarr as well?

**Comment by chrisbenincasa** [comment]
Hey sorry for the delay here - still getting used to the discussion/issue split. I think intermittent watermarks would be a cool feature. I'm pretty sure ETV supports these as "fade points" (assuming I understand the feature correctly). I love the idea of implementing features to replicate that "feel" of TV. 

I think multiple watermarks would be doable, but it's not something I've thought about or considered before. At the moment I'd say that intermittent / timed watermarks would be a more likely feature we'd implement.

**Reply by avalonMMXXII** [reply]
Thank you for taking this into consideration! Yes it would work like a fade in/fade out every certain amount of minutes for whatever desired length of time.

Multiple watermarks perhaps in the future would be something but I mention them because some channels use them (even if it says something simple at the beginning of a TV show/movie saying this program broadcasts in HD or 4K. Some can be used as a TV rating that shows up.

I am glad you are considering the intermittent watermarks though!

**Reply by chrisbenincasa** [reply]
Ahhh i see! Like a "temporary" watermark that could show rating or something. I remember those in the top corner saying TV-14 (in the US)

I've opened https://github.com/chrisbenincasa/tunarr/issues/492 but feel free to open another one there about the temporary watermark idea. We have a few things on our plate that are likely higher priority than this, but I think it's a cool feature and we'll definitely get to it.

**Comment by chrisbenincasa** [comment]
Now tracked in #492
